Gruntfile.js

```diff
@@ -1,21 +1,129 @@
 module.exports = function( grunt ) {
 
 grunt.loadNpmTasks( "grunt-contrib-jshint" );
+grunt.loadNpmTasks( "grunt-contrib-connect" );
+grunt.loadNpmTasks( "grunt-selenium-server" );
 grunt.loadNpmTasks( "grunt-html" );
 grunt.loadNpmTasks( "grunt-jscs" );
 
+var target,
```

Comment:
before this `Gruntfile` gets out of hand, do you think we could adopt the `grunt.loadTasks('build/tasks')` approach http://www.thomasboyt.com/2013/09/01/maintainable-grunt.html